src/DropdownToggle.js

```diff
@@ -21,22 +33,15 @@ const propTypes = {
    */
   children: PropTypes.func.isRequired,
 };
+
 function DropdownToggle({ children }) {
```

Comment:
is this essentially soft-deprecated in favor of the hook now?

Reply:
maybe, i'm not convinced the pure hook api here is the best option, at least not until there is a proper popper.js hooks api